specs/README.md

```diff
@@ -1,63 +1,3 @@
-<!-- DOCTOC SKIP -->
-# Optimism Bedrock specs
+# Specs Have Moved
```

Comment:
At some point Specs will have lived in the new location for long enough that we won't consider them "moved". At that point we could remove the whole `specs` folder. Considering you've already changed the specs pointer in the main README, we could remove this folder any time IMO.

Reply:
Will remove in a month or so I think - there could be links on the web to stuff in that folder, so if people click around I want to make it clear where to go. GitHub doesn't support redirects in repositories, sadly.